src/lib/system-info.ts

```diff
@@ -11,9 +11,9 @@ import { exec } from './fs-utils';
 
 export async function getCpuUsage(): Promise<number> {
 	const cpuData = await systeminformation.currentLoad();
-	const totalLoad = cpuData.cpus.reduce((load, cpuLoad) => {
-		return load + cpuLoad.load;
-	}, 0);
+	const totalLoad = _.sumBy(cpuData.cpus, ({ load }) => {
```

Comment:
This one is debatable because it's using lodash but the lodash usage is pretty prevalent

Reply:
If using lodash would result in fewer computations, then the usage is fine by me!